man/adoc/bpftrace.adoc

```diff
@@ -891,6 +891,21 @@ All valid identifiers can be used as `name`.
 
 The data type of a variable is automatically determined during first assignment and cannot be changed afterwards.
 
+==== Block Expressions
```

Comment:
I think this should go either above comments: https://github.com/bpftrace/bpftrace/blob/master/man/adoc/bpftrace.adoc#comments

Or inside https://github.com/bpftrace/bpftrace/blob/master/man/adoc/bpftrace.adoc#operators-and-expressions

Reply:
Moved to the operators and expressions section